domain/abbreviate.go

```diff
@@ -40,8 +42,21 @@ func (matcher *Matcher) Abbreviate(word string) string {
 
 // Match against a list of mappings
 func (matcher Matcher) Match(word string) string {
-	if abbr, found := matcher.items[word]; found {
+	if abbr, found := matcher.mainWords[word]; found {
 		return abbr
+	} else {
+		for prefix, preAbbr := range matcher.prefixes {
+			if strings.HasPrefix(word, prefix) {
+				return preAbbr + matcher.Match(strings.TrimPrefix(word, prefix))
+
+			}
```

Comment:
Picky, but could you remove the empty space here?

Reply:
Done